.github/workflows/csv-coverage-update.yml

```diff
@@ -0,0 +1,43 @@
+name: Update framework coverage reports
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  update:
+    name: Update framework coverage report
```

Comment:
Add a condition here to make sure it only runs when `github.repository == 'github/codeql'`. Otherwise it will start running on all forks :)

Reply:
In the end I went with `if: github.event.repository.fork == false`, this way I could still test the workflow in [`dsp-testing`](https://github.com/dsp-testing/codeql-csv-coverage-pr-commenter/actions/runs/963385530).

Reply:
Note that `github.event.repository.fork` [is not part of the context](https://github.community/t/github-event-repository-fork-context-does-detect-forks-in-scheduled-jobs/185925) for scheduled github actions. Don't actually know if this matters for this PR, but wanted to give you a heads up in case there's a problem.